src/cdk/v2/destinations/heap/procWorkflow.yaml

```diff
@@ -40,7 +40,9 @@ steps:
       });
       .message.properties.idempotencyKey ? 
         ($.context.payload.idempotency_key = .message.properties.idempotencyKey);
-
+  - name: validateuserId
+    template: |
+      $.assert($.context.payload.identity, "userId is required");
```

Comment:
can we map `anonymousId` too to userID and identity fields during transformation?

Reply:
We are already doing that